src/localization/fr/cheatsheet.json

```diff
@@ -0,0 +1,70 @@
+
+  "cheatsheet.characterClasses": "Classe de caractères",
+  "cheatsheet.characterSet.title": "Jeu de caractères",
+  "cheatsheet.characterSet.description": "Correspond à n'importe quel caractère du jeu.",
+  "cheatsheet.negatedCharacterSet.title": "Jeu de caractères négatifs",
+  "cheatsheet.negatedCharacterSet.description": "Correspond à tout caractère ne faisant pas partie de l'ensemble.",
+  "cheatsheet.range.title": "Plage",
+  "cheatsheet.range.description": "Correspond à tous les caractères entre deux caractères, y compris eux-mêmes.",
+  "cheatsheet.dot.title": "Point",
+  "cheatsheet.dot.description": "Correspond à n'importe quel caractère, sauf les sauts de ligne.",
+  "cheatsheet.word.title": "Mot",
+  "cheatsheet.word.description": "Correspond à tout caractère alphanumérique. Y compris le soulignement.",
+  "cheatsheet.notWord.title": "N'est pas un mot",
+  "cheatsheet.notWord.description": "Correspond à tout caractère non alphanumérique.",
+  "cheatsheet.digit.title": "Chiffre",
+  "cheatsheet.digit.description": "Correspond à tout caractère numérique.",
+  "cheatsheet.notDigit.title": "N'est pas un chiffre",
+  "cheatsheet.notDigit.description": "Correspond à tout caractère non numérique.",
+  "cheatsheet.space.title": "Espace",
+  "cheatsheet.space.description": "Correspond à tout caractère d'espacement.",
+  "cheatsheet.notSpace.title": "Pas d'espace",
+  "cheatsheet.notSpace.description": "Correspond à tout caractère autre qu'un espace.",
+
+  "cheatsheet.flags": "Drapeaux",
+  "cheatsheet.caseInsensitiveFlag.title": "Ignorer le cas",
+  "cheatsheet.caseInsensitiveFlag.description": "Rend l'expression insensible à la casse.",
+  "cheatsheet.globalFlag.title": "Drapeaux globaux",
+  "cheatsheet.globalFlag.description": "Assure que l'expression ne s'arrête pas à la première correspondance.",
+  "cheatsheet.multilineFlag.title": "Multiligne",
+  "cheatsheet.multilineFlag.description": "Si cette option n'est pas activée, le début et la fin de la ligne correspondent au début et à la fin de la chaîne entière. Cela ne fonctionne pas pour chaque ligne individuellement.",
+
+  "cheatsheet.groupAndReferences": "Groupe & Références",
+  "cheatsheet.group.title": "Groupe",
+  "cheatsheet.group.description": "Grouper une expression.",
+  "cheatsheet.reference.title": "Référence",
+  "cheatsheet.reference.description": "Référence une expression groupée.",
+  "cheatsheet.nonCapturingGroup.title": "Groupe de non-capture",
+  "cheatsheet.nonCapturingGroup.description": "Crée un regroupement qui ne peut pas être référencé.",
+
+  "cheatsheet.lookarounds": "Regard autour",
+  "cheatsheet.positiveLookahead.title": "Regard devant Positive",
+  "cheatsheet.negativeLookahead.title": "Regard devant Négatif",
+  "cheatsheet.positiveLookbehind.title": "Regard derrière Positive",
+  "cheatsheet.negativeLookbehind.title": "Regard derrirèe Négatif",
+
+  "cheatsheet.quantifiersAndAlternation": "Qantificateurs et Alternance",
```

Comment:
would modify line 59 to 
  "cheatsheet.quantifiersAndAlternation": "Quantificateurs et Alternance",